### PR TITLE
Refactor/652 account remove delete

### PR DESCRIPTION
**Pull request by escapedcat**
- Switch `remove` account to `delete` account because it was the same code
- `delete` has more functionality but can handle this as well. Extended the tests
- Switched some code from Promise/then to async/await

### CODE REVIEW

File: src/app/screens/Onboard/TestConnection/index.tsx [hunk 1]
```diff
@@ -13,36 +13,44 @@ export default function TestConnection() {
...
-        const { balance: resBalance } = response.balance;
-        const balance =
-          typeof resBalance === "number" ? resBalance : parseInt(resBalance);
+    try {
+      const response = await api.getAccountInfo();
 
-        setAccountInfo({ alias, balance, name });
-      })
-      .catch((e) => {
+      if ("error" in response) {
```
Comment (bumi) [inline src/app/screens/Onboard/TestConnection/index.tsx:32]: when is this the case? doesn't getAccountInfo throw an error in the error case?

Comment (escapedcat) [reply src/app/screens/Onboard/TestConnection/index.tsx]: Yeah, thanks. Had a look again and changed it.

File: src/app/screens/Onboard/TestConnection/index.tsx [hunk 2]
```diff
@@ -13,36 +13,39 @@ export default function TestConnection() {
...
+      const { balance: resBalance } = response.balance;
+      const balance =
+        typeof resBalance === "number" ? resBalance : parseInt(resBalance);
 
-        setAccountInfo({ alias, balance, name });
-      })
-      .catch((e) => {
+      setAccountInfo({ alias, balance, name });
+    } catch (e) {
+      if (typeof e === "string") {
```
Comment (dylancom) [inline src/app/screens/Onboard/TestConnection/index.tsx:39]: In which case will a string be thrown?

Comment (escapedcat) [reply src/app/screens/Onboard/TestConnection/index.tsx]: Good point. Checked how we do it in other places and adjusted.

**Review by dylancom — APPROVED** [review]

**Comment by im-adithya** [comment]
@escapedcat ig you forgot to delete the `remove.js` file :P

**Pull request merged**